src/guides/v2.3/extension-dev-guide/framework/float-comparator.md

```diff
@@ -0,0 +1,50 @@
+---
+group: php-developer-guide
+title: Float Comparator
+contributor_name: Adarsh Manickam
+contributor_link: https://github.com/drpayyne
+---
+
+## Overview
+
+The [`Magento\Framework\Math\FloatComparator`]({{ site.mage2bloburl }}/{{ page.guide_version }}/lib/internal/Magento/Framework/Math/FloatComparator.php){:target="\_blank"} library provides methods to compare float values with a proper epsilon value (`0.00001`).
```

Comment:
I think it might be a good idea to explain what Epsilon is and why this method of comparison is more desirable than just saying IS a > b?

Perhaps a link to a page like : https://floating-point-gui.de/errors/comparison/ ?

Everything else is good on this PR

Reply:
Sure, @BarnyShergold! I shall work on that.